bug - TIR is not read-only to all after publish (#23368)
demisto/content: Packs/ThreatIntelReports/ReleaseNotes/1_0_7.md

#### Scripts
##### PublishThreatIntelReport

> - Fixed an issue where publishing a threat intel report did not make it read-only to all roles in the system.

Reviewer: ```suggestion
- Fixed an issue where a published Threat Intel report was not available for users with read-only roles.
```
please make sure I understood the case correctly :)
